### ISSUE TITLE
Could not compile on Fedora 33

### ISSUE THREAD

**Issue by sobotkami** [Doomsdayrs/kotlinx-gtk#1]
The Fedora has the libraries and header of GTK in other directories as the debian based distros have.

How to reproduce:
- get a fresh clone of repository
- run `./gradlew :demos:widget-factory:linkReleaseExecutableNative`

Error:
```
> Task :nativex:cinteropGtkNative FAILED
Exception in thread "main" java.lang.Error: /usr/include/glib-2.0/glib/gtypes.h:32:10: fatal error: 'glibconfig.h' file not found
```

**Comment by sobotkami** [comment]
It can be fixed with following patch:
```
git diff
diff --git a/nativex/src/nativeInterop/cinterop/gtk.def b/nativex/src/nativeInterop/cinterop/gtk.def
index b796e98..3a4f918 100644
--- a/nativex/src/nativeInterop/cinterop/gtk.def
+++ b/nativex/src/nativeInterop/cinterop/gtk.def
@@ -9,7 +9,7 @@ compilerOpts.linux = \
     -I/usr/include/at-spi-2.0 \
     -I/usr/include/dbus-1.0 \
     -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include \
-    -I/usr/include/gtk-3.0 \
+    -I/usr/lib64/dbus-1.0/include \
     -I/usr/include/gio-unix-2.0 \
     -I/usr/include/cairo \
     -I/usr/include/pango-1.0 \
@@ -25,11 +25,13 @@ compilerOpts.linux = \
     -I/usr/include/libmount \
     -I/usr/include/blkid \
     -I/usr/include/glib-2.0 \
-    -I/usr/lib/x86_64-linux-gnu/glib-2.0/include
+    -I/usr/lib/x86_64-linux-gnu/glib-2.0/include \
+    -I/usr/lib64/glib-2.0/include
 
 # pkg-config --libs gtk+-3.0
 linkerOpts.linux =\
     -L/usr/lib/x86_64-linux-gnu/ \
+    -L/usr/lib64/ \
     -lgtk-3 \
     -lgdk-3 \
     -lpangocairo-1.0 \
```

**Comment by Doomsdayrs** [comment]
Fixed with 79cbc8ea
